src/Controls/tests/Xaml.UnitTests.ExternalAssembly/Gh16923Library.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.Maui.Controls.ControlGallery
+{
+	public static class Gh16923Library
+	{
+		public const string LibraryConstant = nameof(LibraryConstant);
+
+		public static class NestedIconCodes
+		{
+			public const string LibraryIconCode = nameof(LibraryIconCode);
```

Comment:
Should we test an `[InternalsVisibleTo]` nested class, too? (not sure if we have a class library setup for that yet)

Reply:
Thanks, I've simplified the xaml in the test a bit and added an internal nested class to the test.